code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -167,10 +167,10 @@
 
 /datum/reagent/drug/methamphetamine/on_mob_add(mob/living/L)
 	..()
-	L.add_trait(TRAIT_GOTTAGOREALLYFAST, id)
+	L.add_trait(TRAIT_IGNORESLOW, id)
 
 /datum/reagent/drug/methamphetamine/on_mob_delete(mob/living/L)
-	L.remove_trait(TRAIT_GOTTAGOREALLYFAST, id)
+	L.remove_trait(TRAIT_IGNORESLOW, id)
 	..()
 
 /datum/reagent/drug/methamphetamine/on_mob_life(mob/living/carbon/M)
```

Comment:
You should change the var/high_message, it can be misleading. 
Also will this be updated on are wiki?